### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Jaymelfah**
#### In this branch,  I used the ERD diagram to create the data model

> Created a new blog app
> Made sure that the project has Postgres database set up.
> Built a schema for the project

### CODE REVIEW

File: db/migrate/20230123192359_add_user_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (t-yanick) [inline db/migrate/20230123192359_add_user_ref_to_posts.rb]: - [ ] When you check the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram,you notice that in the `Post` table, we have `AuthorID`. It therefore means that there's an `Author` table, that links to the `User` table as an alias. I therefore think that your reference should be:
- `:post, :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20230123192710_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (t-yanick) [inline db/migrate/20230123192710_add_user_ref_to_likes.rb]: - [ ] Following the same approach as for the `Post` table, `Author` table is an alias to `User` table and the link should be to the `AuthorId` in the `Author` table. I think you should have:
- `:likes, :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20230123192830_add_user_ref_to_comments.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (t-yanick) [inline db/migrate/20230123192830_add_user_ref_to_comments.rb]: - [ ] You need to create `AuthorId` which is the foreign key coming from the `Author` table which is an alias to the `User` table. I think you should instead have:
- `:comments, :author, foreign_key: { to_table: 'users' }`

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Jaymelfah , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Passed linters
:heavy_check_mark: Descriptive ReadMe file

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalising the first letters of your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable

File: db/migrate/20230123192359_add_user_ref_to_posts.rb (hunk 1 above)
Comment (rloterh) [reply db/migrate/20230123192359_add_user_ref_to_posts.rb]: well resolved! ✔️

File: db/migrate/20230123192710_add_user_ref_to_likes.rb (hunk 2 above)
Comment (rloterh) [reply db/migrate/20230123192710_add_user_ref_to_likes.rb]: well resolved! ✔️

File: db/migrate/20230123192830_add_user_ref_to_comments.rb (hunk 3 above)
Comment (rloterh) [reply db/migrate/20230123192830_add_user_ref_to_comments.rb]: well resolved! ✔️

File: README.md [hunk 4]
```diff
@@ -1,24 +1,165 @@
...
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (rloterh) [inline README.md:163]: - [ ] OPTIONAL
Kindly endeavor to remove the guide note below the MIT license.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Jay,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Migration query properly implemented!
- Good ReadMe!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**